cypress/integration/login.js

```diff
@@ -8,20 +8,25 @@ describe('Sourcebooks login', function() {
         cy.get('.Select.not-valid').should('be.visible')
     })
 
-    it('Should be able to login with role User', function () {
+    it('Should be able to login with role Team Lead', function () {
 
         cy.visit('/');
         cy.get('[id="loginForm.userId"]').click({force:true});
-        cy.get('[aria-label="Demo User"]').click();
+        cy.get('[aria-label="Ieva Stonkaitė"]').click();
         cy.get('[id="loginForm.role"]').click({force:true});
-        cy.get('[aria-label="User"]').click();
+        cy.get('[aria-label="Team Lead"]').click();
         cy.get('[type="submit"]').click();
 
         cy.url().should('include', '/time-logging');
         cy.get('.page__title').contains('Timesheets')
         cy.get('.calendar').should('be.visible')
         cy.get('.tile.form').should('be.visible')
-        cy.get('.user-info__title').contains('Demo User');
-        cy.get('.main-nav').find('li').should('have.length', 1);
+        cy.get('.user-info__title').contains('Ieva Stonkaite');
+        cy.get('.main-nav').find('li').should('have.length', 2);
     })
+
+    it('Should show calendar date', function () {
+        cy.get('.calendar--selected').find('span').contains(1);

```

Comment:
this check selected day, this would not work if another day would be selected. For maximum swag point i suggest asserting that selected day is today's date :)